utils/markdown/index.js

```diff
@@ -20,3 +23,8 @@ export function formatWithRenderer(text, renderer) {
 
     return marked(text, markdownOptions);
 }
+
+export function stripMarkdown(text) {
+    const removeMarkdown = new RemoveMarkdown();
```

Comment:
Do we encounter any measurable overhead by constructing a new `RemoveMarkdown` each time, compared to the single instance before?

Reply:
There's no metrics to compare but I missed that should be single instance.  I'll update this.

Reply:
Done update.